index.html

```diff
@@ -387,6 +392,24 @@ <h3>
         </p>
       </section>
     </section>
+    <section id="feature-policy" data-cite="feature-policy">
+      <h2>
+        Feature Policy integration
+      </h2>
+      <p>
+        This specification defines a policy-controlled feature identified by
+        the string "<code><dfn data-lt="web-share" data-nodefault=
+        "">web-share</dfn></code>". Its <a>default allowlist</a> is
+        '<code>self</code>'.
```

Comment:
Not sure you need all those data-* attributes... the following should work fine.   

```suggestion
        the string "<code><dfn>web-share</dfn></code>". Its <a>default allowlist</a> is
        '`self`'.
```